src/rdbms/azext_rdbms/_help.py

```diff
@@ -31,153 +22,17 @@ def add_helps(command_group, server_type):
                             --sku-name B_Gen4_2 --ssl-enforcement Disabled \\
                             --storage-size 51200 --tags "key=value" --version {{server-version}}
                 """.format(server_type, command_group)
-    helps['{} server restore'.format(command_group)] = """
+    helps['{} server georestore'.format(command_group)] = """
                 type: command
-                short-summary: Restore a server from backup.
+                short-summary: Georestore a server from backup.
                 examples:
-                    - name: Restore 'testsvr' as 'testsvrnew'.
-                      text: az {0} server restore -g testgroup -n testsvrnew --source-server testsvr --restore-point-in-time "2017-06-15T13:10:00Z"
-                    - name: Restore 'testsvr2' to 'testsvrnew', where 'testsvrnew' is in a different resource group than the backup.
+                    - name: Georestore 'testsvr' as 'testsvrnew'.
+                      text: az {0} server georestore -g testgroup -n testsvrnew --source-server testsvr -l westus2"
+                    - name: Georestore 'testsvr2' to 'testsvrnew', where 'testsvrnew' is in the same resource group as the original server but in a different location.
                       text: |
-                        az {0} server restore -g testgroup -n testsvrnew \\
-                            -s "/subscriptions/${{SubID}}/resourceGroups/${{ResourceGroup}}/providers/Microsoft.DBfor{1}/servers/testsvr2" \\
-                            --restore-point-in-time "2017-06-15T13:10:00Z"
+                        az {0} server georestore -g testgroup -n testsvrnew \\
+                            -s "/subscriptions/${{SubID}}/resourceGroups/${{ResourceGroup}}/providers/Microsoft.DBfor{1}/servers/testsvr2" -l westus2 --sku-name GP_Gen5_2 "
```

Comment:
There is an extra `"` at the end of the line that looks like should be removed.

Reply:
Removed